contracts/UniswapV2Factory.sol

```diff
@@ -46,29 +45,44 @@ contract UniswapV2Factory is IUniswapV2Factory {
         // assembly {
```

Comment:
Can we remove code which is commented if not used

Reply:
removed